package.json

```diff
@@ -46,7 +47,7 @@
     "rollup": "1.27.8",
     "rollup-plugin-terser": "5.1.2",
     "rollup-plugin-typescript2": "0.25.3",
-    "typescript": "3.7.3"
+    "typescript": "3.9.0-beta"
```

Comment:
I’m hesitant about pulling in a beta for this. I might hold off on merging this until 3.9.0 goes stable.

Reply:
I can comment out the few tests that make use of this right now and enable them after 3.9 is out